frontend/client/vite.config.ts

```diff
@@ -21,9 +21,7 @@ export default defineConfig(({ mode }) => {
         },
         server: {
             port: VITE_PORT,
-            hmr: {
-                port: VITE_PORT,
-            },
+            hmr: false
```

Comment:
Don't merge it like this please, I'll have to revert those changes anyway. 

If you want to disable HMR, then we should use env variables to achieve that. Otherwise we'll just be toggling it on and off.

Reply:
ok, will be reverted